WordPress/src/main/java/org/wordpress/android/ui/reader/services/ReaderUpdateService.java

```diff
@@ -200,9 +200,15 @@ private static ReaderTagList parseTags(JSONObject jsonObject, String name, Reade
             String internalName = it.next();
             JSONObject jsonTopic = jsonTopics.optJSONObject(internalName);
             if (jsonTopic != null) {
-                String tagName = JSONUtils.getStringDecoded(jsonTopic, "title");
+                String tagTitle = JSONUtils.getStringDecoded(jsonTopic, "title");
+                String tagName;
+                if (jsonTopic.has("display_name")) {
```

Comment:
Maybe use `slug` vs `display_name`? It might help clarify the distinction between the tags name and the tags title.  `display_name` is rather unfortunate as a dictionary key. At a glance you'd think its what would be presented to the user on screen.